docs/data/enumerations/district_type.yaml

```diff
@@ -0,0 +1,46 @@
+name: DistrictType
+tags:
+- _name: city
+  description: A city.
+- _name: city-council
+  description: A specific seat/jurisdiction for a city, town, or village council.
+- _name: congressional
+  description: A United States congressional district.
+- _name: county
+  description: A county.
+- _name: county-council
+  description: A county council district, either in its entirety or for a specific
+    seat.
+- _name: judicial
+  description: A judicial district.
+- _name: municipality
+  description: A civil division which is not a town, city, village, or county.
+- _name: national
+  description: The United States.
+- _name: school
+  description: A school district.
+- _name: special
+  description: A `special-purpose district`_ that exist separate from general-purpose
```

Comment:
@cjerdonek I should have phrased it better. What I was trying to get at was the idea of adding some sort of "links" section at the bottom of the YAML, so that way the whole RST content (including the table and the links) would be generated from the YAML. E.g.,

```
links:
  _name: special-purpose district
  url: http://en.wikipedia.org/wiki/Special-purpose_district
  _name: town
  url: http://en.wikipedia.org/wiki/Town#United_States
  _name: Wikipedia article
  url: http://en.wikipedia.org/wiki/Town#United_States
```